src/test/java/org/java_websocket/AbstractWrappedByteChannelTest.java

```diff
@@ -0,0 +1,222 @@
+package org.java_websocket;
+
+import org.java_websocket.AbstractWrappedByteChannel;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.Timeout;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.channels.ByteChannel;
+
+
+
+public class AbstractWrappedByteChannelTest {
+
+  @Rule
+  public final Timeout globalTimeout = new Timeout(10000);
+
+  @Rule
+  public final ExpectedException thrown = ExpectedException.none();
+
+  // Test written by Diffblue Cover
+  @Test
+  public void closeOutputNullPointerException999c2a1e4c5166836ee() throws IOException {
```

Comment:
Hashes???

Reply:
That's what json_to_test outputs

Reply:
We are using this version of the combiner https://github.com/diffblue/fuzztest/blob/develop/tests/regression-tests/bin/test-combiner.js

Reply:
That version of the combiner was designed for a very limited purpose: it just runs the minimum amount of code to produce .java files. Since its only use is to ensure that we produce tests that compile and run, and not to do anything customer-facing, I never implemented the code that would strip hashes.